inventory/Inventory/settings.py

```diff
@@ -162,12 +162,10 @@
 
 STATIC_URL = 'static/'
 
-# STATICFILES_DIRS = [
-#     BASE_DIR / "static",
-# ]
-
 STATIC_ROOT = (BASE_DIR/"assets/")
 
+STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
```

Comment:
Don't push this to production there's a new pull request with the updated version